idle-presentation/src/main/kotlin/com/swm/idle/presentation/chat/config/ChatHandshakeInterceptor.kt

```
@@ -0,0 +1,46 @@
+    override fun beforeHandshake(
+        request: ServerHttpRequest,
+        response: ServerHttpResponse,
+        wsHandler: WebSocketHandler,
+        attributes: MutableMap<String, Any>
+    ): Boolean {
+        if (request is ServletServerHttpRequest) {
+            val accessToken = request.servletRequest.getHeader("Authorization")
+
+            val claims = try {
+                JwtTokenProvider.verifyToken(accessToken, jwtTokenProperties.access.secret).getOrThrow()
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)